server.js

```diff
@@ -1,23 +1,103 @@
 import express from "express";
```

Comment:
Im very impressed with your code. I can't really find anything to give constructive feedback about ... :)